### DISCUSSION TITLE
Nuxt Content 2 and full text search

### DISCUSSION THREAD

**Discussion by lucpotage** [nuxt/content#1125] [Q&A]
Hello! I read in Nuxt Content 2 docs that full text search will not be available. That's one of the reasons I liked v1, being able to search within files and not only file metadata.

Is there a motivation behind this choice? A technical constraint or limited time to implement it?

If it's not planned in the future, any recommendation on what I should use instead (apart from using a db service)?

**Comment by lpheller** [comment]
I would appreciate information on this topic too. I've just picked up Nuxt v3 and content-v2, but the lack of this feature makes me questioning this coice.
I really don't understand why this feature was dropped - unless of course it's just something temporary and will be back in the future. For me, it was an absolute selling point not having to rely on something like Algolia for a small private content project.
Are there any other alternatives?
